snooty/test_parser.py

```diff
@@ -876,7 +876,7 @@ def test_glossary_node() -> None:
         <term>
           <text>$cmd</text>
           <inline_target domain="std" name="term">
-            <target_identifier ids="['term-cmd']">
+            <target_identifier ids="['$cmd']">
```

Comment:
I think it would be problematic for Snooty to generate different IDs than Sphinx tooling for the glossary (Sphinx IDs being `term-{term}`). This is especially relevant given that `mms-docs` may not be migrated. They have 1000+ uses of `:term:` on the master branch alone, which would mean a lot of broken links.

Reply:
I think it should be fine! Targets are exported and imported between properties through intersphinx, which encodes the actual URL. It is probably worth my testing this quickly just to make sure though

Reply:
Ah, good point!

Reply:
I think this is actually fine!

The actual target ID in Sphinx is `$cmd`, NOT `term-cmd`: that's a bug that currently keeps terms from working.

The element ID is fine to change because it's only exposed through the intersphinx inventory, which contains the full url.

Let me know if I'm missing something!